### ISSUE TITLE
Files Stuck on "Queued"

### ISSUE THREAD

**Issue by ReiSixx9** [sc0ty/subsync#55]
I left my computer running with Subsync running for over 12 hours and yet all the files are still on "queued..."

![](https://i.imgur.com/Xt9Q819.png)

**Comment by ReiSixx9** [comment]
> 
> 
> Could you repeat it with logging enabled? https://github.com/sc0ty/subsync/wiki/How-to-dump-logs
> About minute or so after clicking start should be enough.

[Log.txt](https://github.com/sc0ty/subsync/files/4200430/Log.txt)

**Comment by sc0ty** [comment]
Something is wrong with S01E11.ja.srt file, but it still should show error popup when you added this file, or at least when you hit start.
Could you try to synchronize without it?
Also could you please share this file with me so I could try to reproduce this bug myself?
Thanks.

**Comment by ReiSixx9** [comment]
> 
> 
> Something is wrong with S01E11.ja.srt file, but it still should show error popup when you added this file, or at least when you hit start.
> Could you try to synchronize without it?
> Also could you please share this file with me so I could try to reproduce this bug myself?
> Thanks.

Huh, that didn't seem to help unfortunately. Here's the new log and [the .srt file](https://gofile.io/?c=9bIJ7s).
[Log.txt](https://github.com/sc0ty/subsync/files/4200729/Log.txt)

**Comment by sc0ty** [comment]
Ok, I think I know whats going on. I was able to get similar error by adding about 300 files to list. I think the GUI library (wxpython) is struggling to create so much elements. How many files are you trying to synchronize?
Funny thing is it looks like its trying to show error popup, but it fails for the same reason.

**Comment by ReiSixx9** [comment]
> 
> 
> Ok, I think I know whats going on. I was able to get similar error by adding about 300 files to list. I think the GUI library (wxpython) is struggling to create so much elements. How many files are you trying to synchronize?
> Funny thing is it looks like its trying to show error popup, but it fails for the same reason.

Yeah, I'm trying about 350 files. That's probably the problem. Let me try less and see if that fixes it.

**Comment by sc0ty** [comment]
Great!
I will try to add some kind of protection for this that would at least show some error popup and not fail silently.
Thanks for your help debugging this.

**Comment by sc0ty** [comment]
It's still a bug, so I'm reopening it.
I will close it when it will be fixed.
